buildroot/share/PlatformIO/boards/BigTree_SKR_2.json

```diff
@@ -0,0 +1,46 @@
+{
+  "build": {
```

Comment:
This board Json file is also no longer needed.

Reply:
https://github.com/MarlinFirmware/Marlin/pull/21503/commits/460e97748980a6c79e282424e7636f6bb9bdfaf4